### ISSUE TITLE
Juniper, disable parameters ignored under interfaces

### ISSUE THREAD

**Issue by dgarros** [batfish/batfish#3442]
I noticed that the parameter `disable` is ignored when it's configured at the physical interface level

In my configuration I have the following interface
```
et-0/0/48 {
        disable;
        apply-groups CLUSTER-SW-PORT;
        description "description";
        unit 0 {
            family inet {
                address <ipv4>/31;
            }
            family inet6 {
                address <ipv6>/127;
            }
        }
    }
```

and in pybatfish it looks like that
```
(Pdb) int
Interface                               device:et-0/0/48
Access_VLAN                                                   None
Active                                                        True
Allowed_VLANs
All_Prefixes                                                    []
Auto_State_VLAN                                               True
Bandwidth                                                    1e+11
Blacklisted                                                  False
Channel_Group                                                 None
Channel_Group_Members                                           []
Declared_Names                                      [u'et-0/0/48']
Description                       description
DHCP_Relay_Addresses                                            []
Encapsulation_VLAN                                            None
HSRP_Groups                                                     []
HSRP_Version                                                  None
Incoming_Filter_Name                                          None
Interface_Type                                            PHYSICAL
MLAG_ID                                                       None
MTU                                                           1500
Native_VLAN                                                   None
OSPF_Area_Name                                                None
OSPF_Cost                                                     None
OSPF_Enabled                                                 False
OSPF_Hello_Multiplier                                            0
OSPF_Passive                                                 False
OSPF_Point_To_Point                                          False
Outgoing_Filter_Name                                          None
Primary_Address                                               None
Primary_Network                                               None
Proxy_ARP                                                    False
Rip_Enabled                                                  False
Rip_Passive                                                  False
Routing_Policy_Name                                           None
Spanning_Tree_Portfast                                       False
Speed                                                         None
Switchport                                                   False
Switchport_Mode                                               NONE
Switchport_Trunk_Encapsulation                               DOT1Q
VRF                                                        default
VRRP_Groups                                                     []
Zone_Name                                                     None
```
I noticed a similar behavior with `bgp` too, I'll open another issue to track that

**Comment by dhalperi** [comment]
tl;dr: we didn't use to export the physical interfaces at all, because they weren't directly used in the L3 queries that dominate analysis, and we're still trying up some loose ends. As a result, we missed some of these -- thanks for the catch!

**Comment by dgarros** [comment]
Thanks for fixing this one so quickly, much appreciated
